resolwe_bio/processes/workflows/geo_import.py

```diff
@@ -183,29 +185,43 @@ def upload_rna_gse(self, inputs, gse):
                 for srx_id in sample_found:
                     sample_info[srx_id] = name
                     info_file = f"{gse.name}.csv"
-                    run_info = requests.get(
-                        url="https://eutils.ncbi.nlm.nih.gov/Traces/sra/sra.cgi",
-                        params={
-                            "save": "efetch",
-                            "db": "sra",
-                            "rettype": "runinfo",
-                            "term": srx_id,
-                        },
-                    )
-
-                    if run_info.status_code != 200:
```

Comment:
Is there a benefit to excluding checking for return code 200 and just checking for whitespace?

Reply:
We now include the ``raise_for_status()`` which raises an exception for all non-2xx status codes.